### PR TITLE
rename and export parse_dep_string

### PR DESCRIPTION
**Pull request by jeroen**

### CODE REVIEW

**Comment by gaborcsardi** [comment]
Can you pls add a small test case? Thanks!

**Comment by jeroen** [comment]
OK, done (and rebased).

File: test.js [hunk 1]
```diff
@@ -149,3 +149,10 @@ test.cb('parse_file, zip file', function(t) {
 	t.end();
     });
 });
+
+test.cb('parse dependency string', function(t){
+    var deps = desc.parse_dep_string("foo (1.2.3), bar");
```
Comment (gaborcsardi) [inline test.js]: Shouln't this have a `>=`?

Comment (jeroen) [reply test.js]: Yeah that works too. For this test I just did an exact version. Should I change it?

Comment (jeroen) [reply test.js]: OK, updated :)

Comment (gaborcsardi) [reply test.js]: > Yeah that works too. For this test I just did an exact version. Should I change it?

But is that even allowed in R?

Comment (jeroen) [reply test.js]: Not for version requiements no. I use `pkg (1.2.3)` syntax for storing which system dependencies were used to check. I updated the test with a better example.

**Comment by gaborcsardi** [comment]
Thanks!

**Pull request merged**